Integrations/integration-GRR.yml

```diff
@@ -34,8 +34,30 @@ script:
     var server = params.server + ':' + params.port;
     var insecure = params.insecure;
 
+    var getCSRFToken = function(url) {
+        var res = http(
+                url,
+                {
+                    Method: 'GET',
+                    Username: username,
+                    Password: password
+                },
+                insecure
```

Comment:
And what about proxy? (was requested in the other PR)

Reply:
@orius123 - GRR is an incident response tool that is typically installed within an organization's internal infrastructure.  Additionally, I don't see any references on the GRR documentation for allowing GRR clients to proxy IR data back to GRR server.

Reply:
if it's always in the same network, and no way you will need proxy to access the GRR server, than it's cool, but if in some cases you might need it, why not having it here? 
don't see how it can hurt us.. @anara123 WDYT?

Reply:
I think it is always better to have it there just in case

Reply:
Sorry for delay... Added proxy @anara123